apps/ledger-live-desktop/src/renderer/screens/asset/AssetBalanceSummaryHeader.jsx

```diff
@@ -98,16 +103,36 @@ export default function AssetBalanceSummaryHeader({
       return pairs && pairs.find(({ from, to }) => [from, to].includes(currency.id));
     });
 
-  const onBuy = useCallback(() => {
-    setTrackingSource("asset header actions");
-    history.push({
-      pathname: "/exchange",
-      state: {
-        mode: "onRamp",
-        currencyId: currency.id,
-      },
-    });
-  }, [currency, history]);
+  const onBuySell = useCallback(
+    (mode = "buy") => {
```

Comment:
is there a reason to make this "mode" a parameter as it's only used one 🤔  (that would simplify things a bit and allow to directly pass the callback and not create inlined fn)